modin/pandas/test/test_series.py

```diff
@@ -396,16 +396,13 @@ def test___gt__(data):
     inter_df_math_helper(modin_series, pandas_series, "__gt__")
 
 
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test___int__(data):
-    modin_series, pandas_series = create_test_series(data)
-    try:
-        pandas_result = int(pandas_series[0])
-    except Exception as err:
-        with pytest.raises(type(err)):
-            int(modin_series[0])
-    else:
-        assert int(modin_series[0]) == pandas_result
+@pytest.mark.parametrize("count_elements", [0, 1, 10])
+@pytest.mark.parametrize("converter", [int, float])
+def test___int__and__float__(converter, count_elements):
```

Comment:
I would rather we test int and float methods separately. We have a test for bool separately as well.

Reply:
Maybe add a bool here and name the test something like `test_converters`? Why split this test?

Reply:
I prefer to have one test per one function. Don't you think this is a good practice?

Reply:
I prefer to separate them.

Reply:
done